modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/util/KBDropdownItemsProvider.java

```diff
@@ -910,6 +915,35 @@ private UnsafeConsumer<DropdownItem, Exception> _getRSSActionUnsafeConsumer(
 		};
 	}
 
+	private UnsafeConsumer<DropdownItem, Exception>
+		_getViewChildArticlesActionUnsafeConsumer(KBArticle kbArticle) {
```

Comment:
This method name is OK, but to make the other similar method name consistent I would rename `_getAddChildActionUnsafeConsumer` to `_getAddChildArticleActionUnsafeConsumer`